### PR TITLE
dynamic insertion of nodes in a linked list in javascript

### PR DESCRIPTION
**Pull request by TidbitsJS**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Created dynamic insertion of nodes in a linked list

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2519

### CODE REVIEW

**Comment by TidbitsJS** [comment]
Kindly review the PR @HarshCasper
Part of GSSOC'21

File: JavaScript/ds/DynamicLinkedList.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+      break;
+
+    default:
+      console.log("You know this is wrong, Bbye!\n".wrong);
+      return;
+  }
+} while (choice !== 0);
+
+
+// Sample I/O of the above program -
```
Comment (HarshCasper) [inline JavaScript/ds/DynamicLinkedList.js:163]: Use only one sample I/O

File: JavaScript/ds/DynamicLinkedList.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+Follow the instructions to get in the show
+0. To exit without doing anything
+1. Display the Linked List
+2. Append element in the Linked List
+3. Prepend element in the Linked List
+4. Insert anywhere in between the Linked List
+Enter your choice - 0
+Hmm, matter of choice!
+
+*/
```
Comment (HarshCasper) [inline JavaScript/ds/DynamicLinkedList.js]: Add a new line at the EOF

File: JavaScript/ds/DynamicLinkedList.js [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+// Instantiating the Linked List class
+const myLinkedList = new LinkedList(10);
+
+myLinkedList.append(5); // Output - [10, 5]
+myLinkedList.append(16); // Output - [10, 5, 16]
+myLinkedList.prepend(2); // Output - [2, 10, 5, 16]
+myLinkedList.insert(2, 56); // Output - [2, 56, 10, 5, 16]
+myLinkedList.insert(1, 22); // Output - [22, 2, 56, 10, 5, 16]
+myLinkedList.insert(myLinkedList.length, 74); // Output - [22, 2, 56, 10, 5, 74, 16]
+
```
Comment (HarshCasper) [inline JavaScript/ds/DynamicLinkedList.js]: Remove this

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: JavaScript/ds/DynamicLinkedList.js (hunk 3 above)
Comment (TidbitsJS) [reply JavaScript/ds/DynamicLinkedList.js]: I made the changes @HarshCasper

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**